src/components/scenes/Loans/LoanManageScene.tsx

```diff
@@ -236,24 +243,40 @@ export const LoanManageScene = (props: Props) => {
             })
           }
           break
-        default: {
+        case 'loan-manage-repay':
           actionOp = {
             type: 'seq',
             actions: [
               {
-                type: actionOpType,
+                type: 'loan-repay',
                 borrowPluginId,
                 nativeAmount: actionNativeAmount,
                 walletId: borrowEngineWallet.id,
                 tokenId: selectedAsset.tokenId
               }
             ]
           }
-        }
+          break
+        case 'loan-manage-withdraw':
+          actionOp = {
+            type: 'seq',
+            actions: [
+              {
+                type: 'loan-withdraw',
+                borrowPluginId,
+                nativeAmount: actionNativeAmount,
+                walletId: borrowEngineWallet.id,
+                tokenId: selectedAsset.tokenId
+              }
+            ]
+          }
+          break
+        default:
+          throw new Error(`LoanManageScene cannot generate actionOps for ${loanManageAction}`)
```

Comment:
The reason I don't like default cases like this is that _today_ they're not needed and we're making assumptions about _future behavior_. I don't like it one bit because we should give the developer the opportunity to make the design decision when (**or if**) that future code change comes.

The right thing to do is use the right tool to help us with this; the static type checker.